stdlib/Distributed/src/Distributed.jl

```diff
@@ -76,7 +76,7 @@ function _require_callback(mod::Base.PkgId)
         # broadcast top-level (e.g. from Main) import/using from node 1 (only)
         @sync for p in procs()
             p == 1 && continue
-            @sync_add remotecall(p) do
+            Base.@async_nowrap remotecall_wait(p) do
                 Base.require(mod)
                 nothing
             end
```

Comment:
If I understand correctly, a simpler approach to this PR (in particular, the error unwrapping type) is to just tag each expression additionally with `@async`:
```suggestion
            @async @async_add remotecall(p) do
                Base.require(mod)
                nothing
            end
```

This is potentially only relevant for `remotecall_eval`, but just wanted to offer it as a thought.

Reply:
not sure if this would work. If I get it right, the Future from remotecall would get to the Channel that holds stuff that `@sync` will wait for, which is precisely what was causing the slowness in the first place. The extra `@async` helps a tiny bit by _sending_ the stuff more asynchronously, but the actual big problem that happens in `sync_end` when _receiving_ the replies is IMO not improved upon here.

(PS it is `@sync_add` right? I hope I didn't miss some magical `@async_add` that does it right, but grep shows nothing :D )